pandas/core/arrays/timedeltas.py

```diff
@@ -172,6 +177,9 @@ class TimedeltaArray(dtl.DatetimeLikeArrayMixin, dtl.TimelikeOps):
 
     _typ = "timedeltaarray"
     _scalar_type = Timedelta
+    _recognized_scalars = (timedelta, np.timedelta64, Tick)
```

Comment:
at some point can u add some comments for each of the properties / methods here (and in all the subclasses)

eg it seems obvious was _is_recognized_scalar is now in this PR but best to have some comments

Reply:
will comment in the next pass